### PR TITLE
User Impersonation

### PR DESCRIPTION
**Pull request by gibsonbailey**
# @ mention of reviewers
@ckcollab  @Logan-Ruf 

# A brief description of the purpose of the changes contained in this PR.
Allows a superuser to login as any other user through the use of the [django-su](https://github.com/adamcharnock/django-su) python package.

# A checklist for hand testing
- [ ] login to django admin as a superuser
- [ ] navigate to `Users` list
- [ ] click on any other user
- [ ] click `Login As` in the top right-hand corner

# Checklist
- [x] Code review by me 
- [x] Hand tested by me 
- [x] I'm proud of my work
- [x] Code review by reviewer
- [ ] Hand tested by reviewer
- [ ] Ready to merge

### CODE REVIEW

File: src/apps/profiles/admin.py [hunk 1]
```diff
@@ -1,5 +1,12 @@
 from django.contrib import admin
+
 from .models import User
 
 
-admin.site.register(User)
+class UserAdmin(admin.ModelAdmin):
+    # The following two lines are needed:
```
Comment (ckcollab) [inline src/apps/profiles/admin.py:7]: needed for what? what does this stuff do? the suspense is killing me!

Comment (gibsonbailey) [reply src/apps/profiles/admin.py]: Psh. Outdated.

Comment (gibsonbailey) [reply src/apps/profiles/admin.py]: image

**Review by ckcollab — APPROVED** [review]
looks good so far, gravy with nice link in menu 🥇

**Pull request merged**